### ISSUE TITLE
When right is disabled, right is not really disabled

### ISSUE THREAD

**Issue by teuneboon** [jakiestfu/Snap.js#18]
Tested this in the example: http://jakiestfu.github.io/Snap.js/demo/apps/rightDisabled.html which ran 1.6.1 at the time of my test.

How to reproduce:
1. Open http://jakiestfu.github.io/Snap.js/demo/apps/rightDisabled.html in Chrome
2. Open the drawer on the left
3. Drag the main #content as far to the left as you can

Expected behavior: dragging stops when the rightmost point of the #content comes into view again

Actual behavior: you can drag to the left as far as you can, and it will show the right drawer, even though it's disabled.

**Comment by riasvdv** [comment]
I solved this by putting minPosition (or maxPosition depending on which side you disabled) at 0 in the settings, this way the side bounces back nicely.

**Comment by teuneboon** [comment]
@Rias500 Seems to solve most of the issue, thanks. Do you know of a way to disable being able to drag there entirely? (so the snap back wouldn't even be needed)

**Comment by jakiestfu** [comment]
@Rias500 `minPosition` and `maxPosition` would best be used to change the width of your drawers. `disable` should do it's job, appropriately. I'll look into this fix and let you know

**Comment by jakiestfu** [comment]
@teuneboon What do you mean disabling drag their entirely? Do you mean disabling the hyperextending drawer? I.E. your drawer width is `266px`, you drag it out to `270px` and there is resistance. Are you talking about having it stop completely at `266px`?

**Comment by teuneboon** [comment]
@jakiestfu I think it's related to the disable thing your already looking into. But:

What I mean is that when I have 'right' disabled and the minPosition set I can still drag my drawer so that it shows the background on the right, even though it snaps back correctly. I attached a screenshot to illustrate what I mean: the part I surrounded with red can be seen even when I have right disabled and minPosition on 0.
![screenshot](https://f.cloud.github.com/assets/571486/426248/53311baa-adc4-11e2-8ae8-610e7e743d27.png)
